package.json

```diff
@@ -10,8 +10,8 @@
         "test": "./node_modules/.bin/jest; exit 0;",
         "lint": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\"; exit 0;",
         "errors": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" --quiet; exit 0;",
-        "dev": "npm ci && webpack --progress --config ./webpack/webpack.dev.config.js",
-        "prod": "npm ci && webpack --progress --config ./webpack/webpack.prod.config.js",
+        "dev": "node ./scripts/buildSitemap.js prod && webpack --progress --config ./webpack/webpack.dev.config.js",
```

Comment:
Do you want `prod` on this line starting with `dev:`?

Maybe so if there's only one way (the way with `prod`) to generate these sitemaps.

Reply:
Yeah... we shouldn't really need the parameter `prod` after the path to the script. This is left over from my initial attempt where we were building per-env sitemaps.